src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -333,43 +314,44 @@ public void Dispose()
             }
         }
 
-        private void EnumeratePSScriptAnalyzerCmdlets()
+        /// <summary>
+        /// Write a list of all PSSA cmdlets that are available to the logs.
+        /// </summary>
+        private void LogAvailablePssaCmdlets()
         {
-            if (_hasScriptAnalyzerModule)
-            {
-                var sb = new StringBuilder();
-                var commands = InvokePowerShell(
+            var commands = InvokePowerShell(
```

Comment:
In a separate PR I'd like to change this from Get-Command to Get-Module -Name PSScriptAnalyzer.  From the resulting PSModuleInfo, we can get the Version of the module as well as the `ExportedCmdlets`.  Unless you're feeling spunky and want to make this change.  :-)